controllers/UserController.js

```diff
@@ -35,7 +35,7 @@ exports.addSDGs = async (req, res, next) => {
                 const user_id = req.params.userId
                 const sdg_ids = req.body.sdg_ids
                 await userService.addSDGs(user_id, sdg_ids);
-                res.send({});
+                res.send({success: true});
```

Comment:
why this ? If the the http status code is 200, the request is successful. I'd like to have to consistent response bodies across the API, to return an object if needed otherwise an empty one